GeneratorInterface/GenFilters/interface/MCParticlePairFilter.h

```diff
@@ -38,6 +38,9 @@
 namespace edm {
   class HepMCProduct;
 }
+namespace HepMC {
```

Comment:
Hi @echapon - i will try to look tomorrow, something is not right in what you see.